common/components/stores/NavigationStore.js

```diff
@@ -10,15 +10,18 @@ import Section from '../enums/Section.js';
 export type NavigationActionType = {
   type: 'SET_SECTION',
   section: SectionType,
+  // page: number,
```

Comment:
Are any of the changes on this page needed?

Reply:
Oh, I need to remove those comments.  These are related to removing the page query parameter from the URL